snuba/datasets/querylog.py

```diff
@@ -0,0 +1,81 @@
+from snuba import settings
+from snuba.clickhouse.columns import (
+    Array,
+    ColumnSet,
+    DateTime,
+    Enum,
+    Float,
+    LowCardinality,
+    Mapping,
+    Nested,
+    Nullable,
+    String,
+    UInt,
+    UUID,
+)
+from snuba.datasets.dataset import Dataset
+from snuba.datasets.dataset_schemas import DatasetSchemas
+from snuba.datasets.querylog_processor import QuerylogProcessor
+from snuba.datasets.schemas.tables import MergeTreeSchema
+from snuba.datasets.table_storage import TableWriter, KafkaStreamLoader
+from snuba.query.extensions import QueryExtension
+
+
+class QuerylogDataset(Dataset):
+    def __init__(self) -> None:
+        status_type = Enum([("success", 0), ("error", 1), ("rate-limited", 2)])
+
+        columns = ColumnSet(
+            [
+                ("request_id", UUID()),
+                ("request_body", String()),
+                ("referrer", LowCardinality(String())),
+                ("dataset", LowCardinality(String())),
+                ("projects", Array(UInt(64))),
+                ("organization", Nullable(UInt(64))),
+                ("timestamp", DateTime()),
```

Comment:
Is this the start or finish timestamp ? (probably better calling it accordingly so that we know which one it is)

Reply:
Hmm, I matched this to "timestamp" in the timer (which I think means start)

Reply:
yeah I would still call it start_stimestamp so people don't have to try to understand how timer works to interpret the results.